src/test/java/nextstep/qna/domain/DeleteHistoriesTest.java

```diff
@@ -0,0 +1,75 @@
+package nextstep.qna.domain;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DeleteHistoriesTest {
+    private static final DeleteHistories D1 = new DeleteHistories();
+    private static final List<DeleteHistory> historyList = new ArrayList<>();
+    @BeforeEach
```

Comment:
개행, 정적 상수 네이밍 컨벤션 확인 부탁드려요 
```suggestion
    private static final List<DeleteHistory> HISTORY_LIST = new ArrayList<>();
    
    @BeforeEach
```